test/sh_spec.py

```diff
@@ -80,8 +80,8 @@ def log(msg, *args):
 
 
 # EXAMPLES:
-## stdout: foo
-## stdout-json: ""
+# stdout: foo
+# stdout-json: ""
```

Comment:
This change looks wrong?

The one below is right.

Reply:
yes my mistake